readme.md

```diff
@@ -27,7 +27,7 @@ Building the App
 First, check out the source, including all the dependencies:
 
 ```
-git clone --recurse-submodules https://github.com/ResearchKit/mPower.git
+git clone --recurse-submodules git@github.com:ResearchKit/mPower.git
```

Comment:
Clone with https is recommended.
souce: https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-https-recommended